core/cli/src/main/java/mil/nga/giat/geowave/core/cli/operations/HelpCommand.java

```diff
@@ -70,90 +66,81 @@ public void execute(
 			lastOperation = entry.getValue();
 		}
 
+		// This is done because if we don't, then JCommander will consider the
+		// given
+		// parameters as the Default parameters. It's also done so that we can
+		// parse prefix annotations and special delegate processing.
+		JCommanderPrefixTranslator translator = new JCommanderPrefixTranslator();
```

Comment:
do we know why the logic here got re-arranged?

Reply:
I don't even think this is relevant to REST services, probably should convert back